aiohue/v2/controllers/groups.py

```diff
@@ -25,6 +25,18 @@ def get_scenes(self, id: str) -> List[Scene]:
         """Get all scenes for this room."""
         return [scene for scene in self._bridge.scenes if scene.group.rid == id]
 
+    def get_lights(self, id: str) -> List[Light]:
+        """Return all lights in given room."""
+        if id not in self._items:
+            return []
+        result = []
+        for dev_id in self._items[id].devices:
+            if dev := self._bridge.devices.get(dev_id):
+                for light_id in dev.lights:
+                    if light := self._bridge.lights.get(light_id):
+                        result.append(light)
```

Comment:
Code can be a bit more readable using guard clauses
```suggestion
            if (dev := self._bridge.devices.get(dev_id)) is None:
                continue

            for light_id in dev.lights:
                if light := self._bridge.lights.get(light_id):
                    result.append(light)
```